src/flows/components/FlowsIndex.tsx

```diff
@@ -61,32 +107,62 @@ const FlowsIndex = () => {
       titleTag={pageTitleSuffixer([PROJECT_NAME_PLURAL])}
       testID="flows-index"
     >
-      <PageHeader fullWidth={false}>
+      <PageHeader
+        fullWidth={false}
+        className={`${showButtonMode && 'withButtonHeader marginHeader'}`}
+      >
         <Page.Title title={PROJECT_NAME_PLURAL} />
+        {showButtonMode && (
+          <>
+            <PresetFlows buttonMode={showButtonMode} />
```

Comment:
why are the concept of "buttonMode" bleeding between these components? since it's already in a boolean conditional, just conditionally render something else